library/lib/core/activity.cpp

```diff
@@ -112,10 +115,26 @@ bool Activity::isHidden()
     return this->contentView->isHidden();
 }
 
-void Activity::registerAction(std::string hintText, enum ControllerButton button, ActionListener actionListener, bool hidden, enum Sound sound)
+ActionIdentifier Activity::registerAction(std::string hintText, enum ControllerButton button, ActionListener actionListener, bool hidden, enum Sound sound)
+{
+    if (this->contentView)
+        return this->contentView->registerAction(hintText, button, actionListener, hidden, sound);
+
+    return ACTION_NONE;
+}
+
+void Activity::unregisterAction(int identifier)
```

Comment:
`int` 😛

Reply:
Sorry, it's taken me so long to fix this. I've had a hell of a week thus far. 😭